.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ commands:
       - crystal/shards-install
       - run: make
       - run: make test
-      - crystal/format-check
+      - run: crystal tool format --check src spec
```

Comment:
Why is this required? It really shouldn't be.

Reply:
See commit message on 4d559accab21a258e372c5c4adb8e0cabfc5ca32

> tmp/integrations is created by integration specs and contains non-formatted files

Reply:
Maybe `crystal/format-check` (or `crystal tool format`) should only apply to `spec src` by default. But this is also a very specific problem that shards contains ill-formatted files in its tree after the test have run.

Alternatives would be to run format check before `make test` or clean `tmp/integrations` afterwards.